vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate/config.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// TODO - make a package that's reusable here and by sharedmain
```

Comment:
FYI, I used sharedmain config set up in my PR: Can you use that?
https://github.com/knative/eventing/pull/2910/files#diff-28e3377805ade46afcad16b416392053R37

Reply:
technically yeah - but that would pull in that package and all it's dependencies

Reply:
that's already part of serving? Or am I confused about something?